src/components/Alert/index.test.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import { render, screen } from '@testing-library/react';
+
+import WarningIcon from 'components/Icon/WarningIcon';
+
+import Alert from '.';
+
+const testAlerData = {
+  dataTestId: 'testAlert',
+  errorMessage: ['test error'],
+  alertHeader: 'Error',
+  icon: 'alert-icon',
+};
+
+describe('Alert', () => {
+  it('render the alert container', () => {
```

Comment:
```suggestion
  it('renders the alert container', () => {
```
Please also check other examples